include/fi_proto.h

```diff
@@ -54,7 +54,8 @@ enum {
 	ofi_ctrl_start_data,
 	ofi_ctrl_data,
 	ofi_ctrl_large_data,
-	ofi_ctrl_ack
+	ofi_ctrl_ack,
+	ofi_ctrl_connect_ack,
```

Comment:
I would rename ofi_ctrl_connect -> ofi_ctrl_connreq, then add connresp.  Ack sounds like it merely acks the message, whereas we really want a full response, which could be a reject message.